include/mbgl/ios/MGLMapView.h

```diff
@@ -1,6 +1,10 @@
 #import <UIKit/UIKit.h>
 #import <CoreLocation/CoreLocation.h>
 
+#import "MGLTypes.h"
```

Comment:
No, I’ve got a patch locally that integrates the user tracking preferences into settings_nsuserdefaults.mm. We need `MGLUserTrackingMode` somewhere generic so that OS X builds don’t have to include this iOS-specific header, which pulls in UIKit.